packages/package_bumbershoot_3_0_8738/.shed.yml

```diff
@@ -1,11 +0,0 @@
-categories:
-- Tool Dependency Packages
-description: Contains a tool dependency definition that downloads the Bumbershoot tools.
-long_description: |
```

Comment:
In the case of Bumbershoot, I don't think these packages were even working for dependency resolution. They never created the proper env.sh for me, and now the tarball they pointed to is gone since I neglected to pin it on TeamCity. The Conda tarball is pinned.